packages/asc-ui/src/components/TopTaskButton/TopTaskButton.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { AnchorHTMLAttributes } from 'react'
+import Icon from '../Icon'
+import TopTaskButtonStyle, { ParagraphStyle } from './TopTaskButtonStyle'
+
+export interface TopTaskButtonProps {
+  icon?: React.ElementType
+  text: string
+  onDarkBackground?: boolean
+}
+
+const TopTaskButton: React.FC<
+  TopTaskButtonProps & AnchorHTMLAttributes<HTMLAnchorElement>
+> = ({ icon, text, ...otherProps }) => {
+  const CustomIcon = icon
+
+  return (
+    <TopTaskButtonStyle {...otherProps} data-testid="root">
```

Comment:
Should we pass a `title` prop?

Reply:
We could, but for what purpose?

Reply:
I'm not sure the `text` prop will always be used as the anchor description, as the CustomIcon could be an element with an `alt` tag right?

Reply:
It should take the contents of the anchor element as it's description, so in this case the text in the body (.e.g the `text` prop). As far as the icon is concerned it is a decorative icon so it should not be read by screen readers.